src/types.ts

```diff
@@ -0,0 +1,34 @@
+import { App } from 'vue'
```

Comment:
what is the difference between `types.d.ts` and `types.ts` .. should be merged, I guess?

Reply:
nope, `*.d.ts` files are used for javascript typings. so the drift API in our case. This should stay separated

Reply:
can we rename it to drift.d.ts` to make this clear?

Reply:
changed in 9697808